code/controllers/subsystems/alarm.dm

```diff
@@ -1,45 +1,56 @@
-
-// We manually initialize the alarm handlers instead of looping over all existing types
-// to make it possible to write: camera_alarm.triggerAlarm() rather than SSalarm.managers[datum/alarm_handler/camera].triggerAlarm() or a variant thereof.
-/var/global/datum/alarm_handler/atmosphere/atmosphere_alarm	= new()
-/var/global/datum/alarm_handler/camera/camera_alarm			= new()
-/var/global/datum/alarm_handler/fire/fire_alarm				= new()
-/var/global/datum/alarm_handler/motion/motion_alarm			= new()
-/var/global/datum/alarm_handler/power/power_alarm			= new()
-
 SUBSYSTEM_DEF(alarm)
 	name = "Alarm"
 	wait = 2 SECONDS
 	priority = FIRE_PRIORITY_ALARM
 	init_order = INIT_ORDER_ALARM
-	var/list/datum/alarm/all_handlers
-	var/tmp/list/currentrun = null
-	var/static/list/active_alarm_cache = list()
 
-/datum/controller/subsystem/alarm/Initialize()
-	all_handlers = list(atmosphere_alarm, camera_alarm, fire_alarm, motion_alarm, power_alarm)
-	. = ..()
+	/// The list of alarm handlers this subsystem processes
+	var/static/tmp/list/datum/alarm_handler/handlers
+
+	/// The list of alarm handlers currently being processed
+	var/static/tmp/list/current = list()
+
+	/// The list of active alarms
+	var/static/tmp/list/active = list()
+
+
+/datum/controller/subsystem/alarm/Initialize(timeofday)
+	handlers = list(
+		GLOB.atmosphere_alarm,
+		GLOB.camera_alarm,
+		GLOB.fire_alarm,
+		GLOB.motion_alarm,
+		GLOB.power_alarm
+	)
+

```

Comment:
```suggestion
	. = ..()
```
thank you Travis

Reply:
Oh dear. Base Initialize is supposed to be a stub. That should be done by master, not by the SS itself. :S

Reply:
Will need to rebase this after #8528